chain/mempool.go

```diff
@@ -238,19 +251,30 @@ func (m *mempool) LoadMempool() error {
 	}
 
 	go func() {
-		for _, txHash := range txs {
-			// Grab full mempool transaction from hash.
-			tx, err := m.client.GetRawTransaction(txHash)
-			if err != nil {
-				log.Warnf("unable to fetch transaction %s for "+
-					"mempool: %v", txHash, err)
-				continue
-			}
+		var eg errgroup.Group
+		eg.SetLimit(runtime.NumCPU())
```

Comment:
This is basically limiting the number of gorotuines that can be executed at the same time in this error group right?

Do we do that because we do not want to saturate the sever with too many `m.client.GetRawTransaction`? Just checking if it makes sense to increase this number to `>NumCPU` while limiting it.

Reply:
Yep, also bitcoind has a default number of rpc threads of 4, so any other requests will block beyond that value.